### DISCUSSION TITLE
Plugin "Marketplace"

### DISCUSSION THREAD

**Discussion by bernhardh** [filamentphp/filament#247] [Ideas]
Sorry for starting another discussion. Hopefully my input isn't anoying.

I saw that there is now the topic "Plugin Development" part of the docs. Cool!

Are there plans, to make something like a "marketplace" for plugins. A lot of admin panels/cms systems has such a thing in one or the other way. For example a build in system like in wordpress or drupal. Or a separate, maintained site like novapackages for nova.

You will never be able to create all fields, widgets, etc. for all purposes by yourself and some "esotheric" fields/widgets/whatever shouldn't be part of the core as well, so why do not "outsource" this to the community?

**Comment by danharrin** [comment]
Yes there are plans to create an official plugin marketplace!

We're also looking to allow plugin authors to charge for their work, which creates a healthy business community around the project as well as funding core development.

**Comment by danharrin** [comment]
Hey! A year later, this is being launched today.

https://filamentphp.com/plugins

Enjoy! ❤️
